### PR TITLE
MAINT: Miscellaneous housekeeping

### PR DESCRIPTION
**Pull request by oesteban**

### CODE REVIEW

File: niworkflows/engine/tests/test_workflows.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    inputnode = Node(niu.IdentityInterface(fields=['in_file']),
+                     name='inputnode')
+    inner = _reorient_wf()
+
+    workflow.connect([
+        (inputnode, inner, [('in_file', 'inputnode.in_file')]),
+    ])
+
+    assert workflow.visit_desc() == "Outer workflow. Inner workflow. Outer workflow (postdesc)."
```
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

**Comment by oesteban** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
- Added 2
           

Complexity increasing per file
==============================
- niworkflows/interfaces/tests/test_utility.py  3
- niworkflows/engine/tests/test_workflows.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911&bid=16766269)

File: niworkflows/engine/tests/test_workflows.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    workflow.connect([
+        (inputnode, deoblique, [('in_file', 'in_file')]),
+        (deoblique, reorient, [('out_file', 'in_file')]),
+        (reorient, outputnode, [('out_file', 'out_file')])
+    ])
+    return workflow
+
+
+def test_boilerplate():
+    """Check the boilerplate is generated."""
```
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [No blank lines allowed after function docstring (found 1)](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

**Comment by oesteban** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911&bid=16766269)

File: niworkflows/engine/tests/test_workflows.py (hunk 1 above)
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

File: niworkflows/engine/tests/test_workflows.py (hunk 2 above)
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [No blank lines allowed after function docstring (found 1)](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

**Comment by oesteban** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911&bid=16766269)

File: niworkflows/engine/tests/test_workflows.py (hunk 1 above)
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

**Comment by oesteban** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
- Added 2
           

Complexity increasing per file
==============================
- niworkflows/engine/tests/test_workflows.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911&bid=16766269)

File: niworkflows/engine/tests/test_workflows.py (hunk 2 above)
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [No blank lines allowed after function docstring (found 1)](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

**Comment by oesteban** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
- Added 2
           

Complexity increasing per file
==============================
- niworkflows/engine/tests/test_workflows.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911&bid=16766269)

File: niworkflows/engine/tests/test_workflows.py (hunk 1 above)
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

File: niworkflows/engine/tests/test_workflows.py (hunk 2 above)
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [No blank lines allowed after function docstring (found 1)](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

File: niworkflows/engine/tests/test_workflows.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
```
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]:
```suggestion
```

**Comment by oesteban** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
- Added 1
           

Complexity increasing per file
==============================
- niworkflows/engine/tests/test_workflows.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911&bid=16766269)

File: niworkflows/engine/tests/test_workflows.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    inputnode = Node(niu.IdentityInterface(fields=['in_file']),
+                     name='inputnode')
+    inner = _reorient_wf()
+
+    workflow.connect([
+        (inputnode, inner, [('in_file', 'inputnode.in_file')]),
+    ])
+
+    assert workflow.visit_desc() == "Outer workflow. Inner workflow. Outer workflow (postdesc)."
```
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

**Comment by oesteban** [comment]
For some obscure reason, it seems codecov is not receiving coverage information from circleci. I think it is related to the fact that I'm following the project in circle (and that was done when it was a fork of poldracklab, not nipreps.

Any ideas @effigies @mgxd ?

Other than that, this is ready for review and merge if reasonable.

**Comment by oesteban** [comment]
I have unfollowed and then followed oesteban/niworkflows from circle. It has added one ssh key, so that might be the culprit. I'll rebase with master and trigger a new build.

**Comment by oesteban** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
- Added 2
           

Complexity increasing per file
==============================
- niworkflows/engine/tests/test_workflows.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911&bid=16766269)

File: niworkflows/engine/tests/test_workflows.py (hunk 2 above)
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [No blank lines allowed after function docstring (found 1)](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

File: niworkflows/engine/tests/test_workflows.py (hunk 1 above)
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

**Comment by oesteban** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
- Added 1
           

Complexity increasing per file
==============================
- niworkflows/engine/tests/test_workflows.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911&bid=16766269)

File: niworkflows/engine/tests/test_workflows.py (hunk 4 above)
Comment (oesteban) [inline niworkflows/engine/tests/test_workflows.py]: Codacy Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911)

**Comment by oesteban** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/nipreps/niworkflows/pullRequest?prid=5096911&bid=16766269)

File: niworkflows/viz/utils.py [hunk 5]
```diff
@@ -22,83 +19,25 @@
...
-            from errno import ENOENT
-            if e.errno == ENOENT:
-                return False
-            raise e
-        return True
+from .. import NIWORKFLOWS_LOG
 
 
 SVGNS = "http://www.w3.org/2000/svg"
 PY3 = version_info[0] > 2
```
Comment (effigies) [inline niworkflows/viz/utils.py]: Might as well drop this and anything that's dependent on it.

**Comment by effigies** [comment]
Woops. Waited to long to hit "Comment"....

**Pull request merged**